WordPress/blogger-importer
# Test plugin for WordPress 6.3

In this PR, we test the plugin for WordPress 6.3 and update old docs URLs.

## Changes

* Test for WordPress 6.3
* Update old documentation URLs

## How to test

1. Clone this PR
2. Using `wp-env` run an instance. The minimum PHP version is 7.0 because WordPress 6.3 [dropped support for PHP 5](https://make.wordpress.org/core/2023/07/05/dropping-support-for-php-5/).
3. Go to `http://localhost:8888/wp-admin/admin.php?import=blogger` and perform an import.
4. Check if the content has been imported and no errors are being generated.
